### DISCUSSION TITLE
Auth Updated Data

### DISCUSSION THREAD

**Discussion by edcelmanuel** [supabase/supabase#1749] [Questions] [answered]
it is possible to access Auth data?

```js
const { user, error } = await supabase.auth.update({ 
  data: { hello: 'world' } 
})
```
Where can i see the user authenticated metadata on supabase dashboard?

**Comment by easy-street** [comment, marked as answer]
i believe the SQL Editor is the best option to view metadata in the dashboard currently. some example queries:

```
select id, raw_user_meta_data from auth.users;
```
![image](https://user-images.githubusercontent.com/64047794/120024650-21ae7800-bfb5-11eb-861c-71fd8aff6ae5.png)

```
select id, hello
from auth.users, jsonb_to_record(raw_user_meta_data) as x("hello" text);
```
![image](https://user-images.githubusercontent.com/64047794/120024719-35f27500-bfb5-11eb-894f-13ae0c1890cf.png)

**Reply by edcelmanuel** [reply]
Oh! nice Thanks you
